book/listings/saving_window_state/1/custom_window/imp.rs

```diff
@@ -0,0 +1,43 @@
+use gio::Settings;
+use glib::signal::Inhibit;
+use gtk::{self};
```

Comment:
```

Reply:
Interesting that I got no warning for that.
Good catch